docs/tutorials/multinode/README.md

```diff
@@ -0,0 +1,332 @@
+```
+The job can be submitted with the `condor_submit` command.
+```
+condor_submit noise.sub
+```
+The output will look as follows:
+```
+Submitting job(s)..................................................
+50 job(s) submitted to cluster 2.
+```
+To monitor the ongoing process of jobs running, you can take advantage of the
+Linux `watch` command and run `condor_q` repeatedly. When complete you watch can
+be exited with cntrl-c.
+```
+watch "condor_q; condor_status"
+```
+The output of this command will show you the jobs in the queue as well as the
+VMs being created to run the jobs. There is a limit of 20 VMs for this
+configuration of the cluster. This can be modified for future runs.
+
+When the command shows the queue is empty, the command can be stopped with cntl-c.
+
+If this is the second _submit_ you have run, you can see the output of the all
+the simualtions. The output will be in the `out` directory. 
+
+```
+cat out/out.2-*
+```
+You can see the results of the simulations.
+```
+Counter({3: 462, 0: 452, 2: 50, 1: 36})
+Counter({0: 475, 3: 435, 1: 49, 2: 41})
+Counter({0: 450, 3: 440, 1: 59, 2: 51})
+Counter({0: 459, 3: 453, 2: 51, 1: 37})
+Counter({3: 471, 0: 450, 2: 46, 1: 33})
+Counter({3: 467, 0: 441, 1: 54, 2: 38})
+Counter({3: 455, 0: 455, 1: 50, 2: 40})
+Counter({3: 466, 0: 442, 2: 51, 1: 41})
+.
+.
+.
+```
+Note that because this is a noise driven circuit, the results of each simulation are different.
+
+## Next steps
+
+To run your own simulations, simply create a noisy circuit in your  _qsim_ python file.
+The file being run in this was `noise3.py`. You can take this example and expand
+to your own circuit. 
+
+```python
+import cirq, qsimcirq
+
+# Create a Bell state, |00) + |11)
+q0, q1 = cirq.LineQubit.range(2)
+circuit = cirq.Circuit(
+    cirq.H(q0),
+    cirq.CNOT(q0, q1),
+    cirq.measure(q0, q1, key='m')
+)
+
+# Constructs a noise model that adds depolarizing noise after each gate.
+noise = cirq.NoiseModel.from_noise_model_like(cirq.depolarize(p=0.05))
+
+# Use the noise model to create a noisy circuit.
+noisy_circuit = cirq.Circuit(noise.noisy_moments(circuit, system_qubits=[q0, q1]))
+
+sim = qsimcirq.QSimSimulator()
+result = sim.run(noisy_circuit, repetitions=1000)
+# Outputs a histogram dict of result:count pairs.
+# Expected result is a bunch of 0s and 3s, with fewer 1s and 2s.
+# (For comparison, the noiseless circuit will only have 0s and 3s)
+print(result.histogram(key='m'))
+```
+
+There are many more examples of circuits to be run [here](https://quantumai.google/cirq/noise)
+
+## FINAL STEP: Shutting down 
+
+> **IMPORTANT**:  To avoid excess billing for this project, it is important to shut down the cluster. This is easily done using the make command built for this purpose.
+```
+make destroy
+```
+
+> The most effective way to ensure you are not charged is to delete the project. [The instructions are here.](https://cloud.google.com/resource-manager/docs/creating-managing-projects#shutting_down_projects)
```

Comment:
Is "ensure you are not charged" accurate?  This procedure does incur some small billing cost even if you delete the project, right?